README.md

```diff
@@ -61,16 +62,20 @@ each file and briefly describe what it is.
 **cases_test.go** - This file contains [generated test cases](#generating-test-cases),
 and will only be present in some exercises.
 
-**example_gen.go** - This file generates the *cases_test.go* file, and will only
-be present in some exercises. See [generating test cases](#generating-test-cases)
-for more information.
-
 **example.go** - This is a reference solution for the exercise.
 
 **leap.go** - This is a *stub file*, and will only be present in some exercises.
 
 **leap_test.go** - This is the main test file for the exercise.
 
+**.meta/** - The *.meta* directory can be used to contain files
+that are not meant to be included when a user fetches an exercise, for instance
```

Comment:
ironically, now that this file says `.meta` isn't included, it may imply (solely by omission) that `example.go` *is* delivered to the student - I suggest it would be good to make clear that `exmaple.go` is also not delivered.